File: backend/SpringBoot/src/main/java/com/myApiLM/SpringBoot/Model/Persona.java

package com.myApiLM.SpringBoot.Model;

import javax.persistence.Basic;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import lombok.Getter;
import lombok.Setter;

@Getter @Setter
@Entity
public class Persona {
    @Id
    @GeneratedValue (strategy = GenerationType.IDENTITY)
    long id;
    @Basic
    String fullName;
    String position;
    String ubication;
    String aboutMe;
    String image;
    String mail;

    public Persona() {
    }
        
    public Persona(long id, String fullName, String ubication, String position, String aboutMe, String image, String mail) {
        this.id = id;
        this.fullName = fullName;
        this.ubication = ubication;
        this.position = position;
        this.aboutMe = aboutMe;
        this.image = image;
        this.mail = mail;
    }


    
 
   
   
}

File: frontend/src/app/entidades/persona.ts
export class Persona {
    id: number;
    fullName: string;
    position: string;
    ubication: string;
    aboutMe: string;
    image: string;
    mail: string;

    constructor(id: number, fullName: string, position: string, ubication: string, aboutMe: string, image: string, mail: string) {
        this.id = id;
        this.fullName = fullName;
        this.position = position;
        this.ubication = ubication;
        this.aboutMe = aboutMe;
        this.image = image;
        this.mail = mail;
    }
}